src/interface.yaml

```diff
@@ -0,0 +1,17 @@
+name: cos_agent
+summary: |
+    This interface allows reactive charms to integrate with operator charms
+    that require `cos_agent` interface, e.g. grafana-agent.
+maintainer: OpenStack Charmers <openstack-charmers@lists.ubuntu.com>
+repo: https://opendev.org/openstack-charmers/charm-interface-cos-agent.git
```

Comment:
There is no `openstack-charmers` organization on opendev.org, the `openstack-charmers` organization lives over on GitHub. The other OVN charm interfaces appear to have lived on opendev.org/x.

However, now that we are moving the charms over to github.com/canonical to get a working CI, I think we should stop putting new things on opendev for this charm.

I think our options are these locations on GitHub:
openstack-charmers/charm-interface-cos-agent.git
canonical/charm-interface-cos-agent.git